base/applications/cmdutils/replace/replace.h

```diff
@@ -0,0 +1,52 @@
+/*
+ * PROJECT:     ReactOS Command shell
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Implements 'replace' command
+ * COPYRIGHT:   Copyright Samuel Erdtman (samuel@erdtman.se)
+ * COPYRIGHT:   Copyright 2024 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
+
+#pragma once
+
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
```

Comment:
```suggestion
#include <stdio.h>
#include <stdlib.h>
#include <tchar.h>

#include <windef.h>
#include <winbase.h>
#include <wincon.h>
#include <winuser.h>
```
Not sure whether the winuser.h is necessary. If not, remove it from this list.